docs/cloud/cloud-org/cloud-org-billing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Manage billing
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 5
+---
+
+# How do I manage billing for my organization?
+
+{: .no_toc }
+
```

Comment:
Need a bit of an explanation to give users an idea of what the page concerns.

For example:

Learn how to manage payments for Database usage before or after your trial expires.